tests/integration/minibroker_test.go

```diff
@@ -292,7 +307,7 @@ var _ = Describe("classes", func() {
 					"Command": []string{
 						"sh", "-c",
 						"redis-cli" +
-						" -u ${DATABASE_URL}",
+							" -u ${DATABASE_URL}",
```

Comment:
Same as `mongo` re: odd arguments to `sh` (here `CLUSTER INFO`).

Reply:
Fixed with 6600dd0.